extra-dev/packages/coq-mi-cho-coq/coq-mi-cho-coq.dev/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+synopsis: "A specification of Michelson in Coq to prove properties about smart contracts in Tezos"
```

Comment:
Would be good to have a multi-line `description` as well, that explains that this contains both a formal definition of the language and it semantics _and_ a weakest-precondition based framework to reason about programs. This improves searchability a lot for people using the archive.

Reply:
OK thanks for the review, I am asking the original authors...

Reply:
Here is a suggestion from the top of my head:
```
description: """
Michelson is a language for writing smart contracts on the Tezos blockchain.
This package provides a Coq encoding of the syntax and semantics of Michelson,
automatically generated by the Ott tool. Also included is a framework called Mi-Cho-Coq
for reasoning about Michelson programs in Coq using a weakest precondition calculus."""
```

Reply:
Thanks for the suggestion and all the fixes, I added the description!